multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py

```diff
@@ -41,14 +41,7 @@ class IConfig(Protocol):
     esdt_contract_address: IAddress
 
 
-class TokenTypes(Enum):
-    NFT = "NFT"
-    SFT = "SFT"
-    META = "META"
-    FNG = "FNG"
-
-
-class RegisterAndSetAllRolesTokenType(Enum):
+class TokenType(Enum):
```

Comment:
This can even be moved to `tokens.py`. But for now, keep it here, all right.

We can define this in the specs (in tokens.md).